src/aiida/manage/configuration/settings.py

```diff
@@ -73,22 +93,16 @@ def get_configuration_directory():
     The location of the configuration directory is defined following these heuristics in order:
 
         * If the ``AIIDA_PATH`` variable is set, all the paths will be checked to see if they contain a
-          configuration folder. The first one to be encountered will be set as ``AIIDA_CONFIG_FOLDER``. If none of them
-          contain one, the last path defined in the environment variable considered is used.
+          configuration folder. The first one to be encountered will be set as ``_glb_aiida_config_folder``.
+          If none of them contain one, the last path defined in the environment variable considered is used.
```

Comment:
```suggestion
```
I need to update the docstring for this function.
The change here is to make the behavior of `get_configuration_directory()` exact counterpart of `set_configuration_directory()` so it can be used as across the code base.